storage/client_test.go

```diff
@@ -51,6 +51,15 @@ import (
 	"github.com/gogo/protobuf/proto"
 )
 
+func rg1(s *storage.Store) client.Sender {
+	return client.Wrap(s, func(ba roachpb.BatchRequest) roachpb.BatchRequest {
+		if ba.RangeID == 0 {
+			ba.RangeID = 1
```

Comment:
I've wanted to do something like this for a long time, although I was planning to look up the range ID based on the keys in the request (kind of like a mini DistSender). We could even put the lookup on the Store itself so we wouldn't have to use a test-specific wrapper.